SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/VisibilityTest/VisibilityTest.cpp

```diff
@@ -79,6 +81,11 @@ void AVisibilityTest::PrepareTest()
 
 				FinishStep();
 			}
+
+			if (AssertTrue(HasAuthority(), TEXT("We should have authority over the test actor.")))
```

Comment:
nit - looks strange having logic run after FinishStep has been called, can we move this to before the IsValid(PC) check?

Reply:
Ah, true, can move it.